modules/edu.gemini.giapi.client/src/main/scala/edu/gemini/giapi/client/gpi/GPIClient.scala

```diff
@@ -3,13 +3,19 @@
 
```

Comment:
This code seems to have the same function for GPI that GmosEpics.scala. Shouldn't it be in seqexec/server/gpi ?

Reply:
Eventually I"ll move it there